tests/unit_tests/iXBRLViewerPlugin/test_xhtmlserialize.py

```diff
@@ -140,6 +140,11 @@ def test_custom_xml_serializer(self):
                 r'''<div attr1='"foo"'>abc</div>''',
                 r'''<div attr1="&quot;foo&quot;">abc</div>''',
             ),
+            # xml prefix doesn't need declaring
+            (
+                r'''<div xml:lang="en" attr='abc'>abc</div>''',
```

Comment:
Should `attr='abc'` be `attr="abc"`? Above testcase implies maybe to me? Also why `r` this doesn't look like a regex?

Reply:
> Should `attr='abc'` be `attr="abc"`

This was copy and paste from the previous test, which is checking that attribute quotes are normalized to double quotes, regardless of contents, but I think it's harmless here, as it's just an equivalent input document.

>  Also why `r` this doesn't look like a regex?

`r` is for any raw string, and avoids having to escape `\`.  It's only actually needed for the two tests that the `style` escaping to unicode escape sequences, but it's harmless to use it for all tests.  Similarly, we only need the `'''` for the tests that mix `'` and `"`, but using it consistently saves having to think and makes the strings line up nicely :)